src/ppx/application.cpp

```diff
@@ -1128,14 +1128,9 @@ void Application::TakeScreenshot()
     unsigned char* texels = nullptr;
     screenshotBuf->MapMemory(0, (void**)&texels);
 
-    std::string filepath;
-    if (mStandardOptions.screenshot_filepath.empty()) {
-        std::string filename = "screenshot_frame" + std::to_string(mFrameCount) + ".ppm";
-        filepath             = (std::filesystem::path(mStandardOptions.screenshot_out_dir) / filename).string();
-    }
-    else {
-        filepath = std::filesystem::path(mStandardOptions.screenshot_filepath).string();
-    }
+    std::string filepath = mStandardOptions.screenshot_path.empty()
+                               ? "screenshot_frame" + std::to_string(mFrameCount) + ".ppm"
+                               : mStandardOptions.screenshot_path;
```

Comment:
Should we add ".ppm" if it's missing?

Reply:
In favor of a no.
-> it adds complexity ("complexity" yes 😊)
-> if the user wants it, they can specify it.
-> this way we don't restrict the filename used, maybe the user wraps a script calling `mktemp()` or another function and just wants us to write the content to this file without extensions?